### PR TITLE
Fix build in gcc7

### PR DESCRIPTION
**Pull request by proller**
also
https://github.com/google/cctz/pull/44

### CODE REVIEW

File: dbms/include/DB/Common/UInt128.h [hunk 1]
```diff
@@ -8,6 +8,11 @@
 namespace DB
 {
 
+#if !__clang__
```
Comment (alexey-milovidov) [inline dbms/include/DB/Common/UInt128.h]: I would like to see it in more appropriate value, for example, around inner loop in Aggregator.cpp, where is the origin of this warning...

Comment (proller) [inline dbms/include/DB/Common/UInt128.h]: Aggregator.cpp already have it -
https://github.com/yandex/ClickHouse/blob/master/dbms/src/Interpreters/Aggregator.cpp#L574

Comment (alexey-milovidov) [inline dbms/include/DB/Common/UInt128.h]: So, I wonder why it is not enough.

Comment (alexey-milovidov) [inline dbms/include/DB/Common/UInt128.h]: Warning suppression here looks sick.
... especially without comment.

**Pull request merged**